Lombiq.NodeJs.Extensions/package.json

```diff
@@ -14,7 +14,7 @@
     "eslint": "8.47.0",
     "if-env": "1.0.4",
     "glob": "10.3.10",
-    "markdownlint": "0.30.0",
+    "markdownlint": "0.34.0", // When upgrading, also upgrade the "markdown-lint" action in Lombiq.GitHub.Actions to a release that uses the same version of markdownlint.
```

Comment:
Definitely don't place comments into the _package.json_ file. It gets overwritten if we ever install another package from the CLI, and right now it breaks if you try to run NPM or PNPM like this:
![image](https://github.com/Lombiq/NodeJs-Extensions/assets/4007293/c3fb418c-cf11-419e-9abf-deeb92ed92de)

Reply:
Dang it, this is what broke the OSOCE build, thanks...